server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java

```diff
@@ -99,6 +99,11 @@
         Setting.timeSetting("cluster.publish.timeout",
             TimeValue.timeValueMillis(30000), TimeValue.timeValueMillis(1), Setting.Property.NodeScope);
 
+    // the interval between reports of slow publication
+    public static final Setting<TimeValue> PUBLISH_REPORT_INTERVAL_SETTING =
```

Comment:
the setting is not registered. Is that by design or omission?

Reply:
Omission :(

We really should have build-time checks for this.